plugins/modules/proxmox.py

```diff
@@ -821,7 +820,7 @@ def build_volume(
                     if new_value not in current_values:
                         update_config = True
                         break
-            # if it is not a list (or string) just compare the current value
+            # if it's not a list (or string) just compare the current value
```

Comment:
Please try to avoid contractions - not everyone reading and using this code is fluent in English, it costs nothing to make their life a bit easier.